runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -73,7 +74,7 @@ public static ExecutionEnvironment createBatchExecutionEnvironment(FlinkPipeline
   static ExecutionEnvironment createBatchExecutionEnvironment(
       FlinkPipelineOptions options, List<String> filesToStage, @Nullable String confDir) {
 
-    LOG.info("Creating a Batch Execution Environment.");
+    LOG.info("Creating a Batch Execution Environment with config {}.", confDir);
```

Comment:
Most log spams comes from `org.apache.flink.runtime` during flink minicluster initiating. Tested that if assign confDir to a directory that has org.apache.flink.runtime WARN logging, it works. However, currently this parameter is always null (seems like beam not respecting parsed in flinkConfDir)